components/widgets/links/notify_link.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import {useIntl} from 'react-intl';
+import {Client4} from 'mattermost-redux/client';
+
+const NotifyLink = (): JSX.Element => {
+    const NOT_STARTED = 'Not_Started';
```

Comment:
maybe we could leverage in the typescript enum struct for declaring these values?